test/integration/use_int_test.go

```diff
@@ -126,6 +127,17 @@ func (suite *UseIntegrationTestSuite) TestReset() {
 	cp.Expect("Checked out project")
 	cp.ExpectExitCode(0)
 
+	// Create a zsh RC file to ensure that the path is updated for both bash and zsh.
+	var zshRcFile string
+	var err error
+	if runtime.GOOS != "windows" {
+		zsh := &zsh.SubShell{}
```

Comment:
This is assuming that the test is ran as "not zsh". Suggest adding `SHELL=bash` to the env to make that assumption explicit.